logexp/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT = google-containers
+IMG = gcr.io/$(PROJECT)/logexp
+TAG = test
+
+.PHONY: build push
+
+all: build
+
+cmd/logexp: cmd/main.go
+	CGO_ENABLED=0 go build -o cmd/logexp k8s.io/test-infra/logexp/cmd
+
+build: cmd/logexp
+	docker build --pull -t $(IMG):$(TAG) cmd
+	docker tag $(IMG):$(TAG) $(IMG):latest
+	rm cmd/logexp
+	@echo Built $(IMG):$(TAG) and tagged with latest
+
+push: build
+	gcloud docker -- push $(IMG):$(TAG)
+	gcloud docker -- push $(IMG):latest
```

Comment:
Why do we push "latest" too? I think we're generally trying to avoid using "latest" tag.
@fejta - is this some common pattern?